README.md

```diff
@@ -349,10 +364,10 @@ near-runner will support expanded patchState-based functionality in the future:
     )
     ```
 
-  - always recreate such data directory instead (the default behavior)
+  * always recreate such data directory instead (the default behavior)
 
-  - specify which port to run on
+  * specify which port to run on
 
-  - and more!
+  * and more!
 
-- escape hatch to `near-api-js` – the `Account` & `ContractAccount` types returned by `getAccount` and `getContractAccount` contain a **`najAccount`** property. This returns an [`Account` type from `near-api-js`](https://near.github.io/near-api-js/classes/account.account-1.html). If you need functionality that `near-runner` doesn't yet provide, this may give you a way to do it. (But please do send a Pull Request adding the functionality directly to `near-runner`!)
+* escape hatch to `near-api-js` – the `Account` & `ContractAccount` types returned by `getAccount` and `getContractAccount` contain a **`najAccount`** property. This returns an [`Account` type from `near-api-js`](https://near.github.io/near-api-js/classes/account.account-1.html). If you need functionality that `near-runner` doesn't yet provide, this may give you a way to do it. (But please do send a Pull Request adding the functionality directly to `near-runner`!)
```

Comment:
Since now we have `accountView()` in `NearAccount` and seems `najAccount` is no longer exposed. This section can be updated as well.

Reply:
Thanks! Removed it!